salt/minion.py

```diff
@@ -2185,12 +2186,12 @@ def matchers_refresh(self):
 
     # TODO: only allow one future in flight at a time?
     @tornado.gen.coroutine
-    def pillar_refresh(self, force_refresh=False):
+    def pillar_refresh(self, force_refresh=False, notify=False):
         '''
         Refresh the pillar
         '''
         if self.connected:
-            log.debug('Refreshing pillar')
+            log.debug('Refreshing pillar. Notify: {}'.format(notify))
```

Comment:
Please use `%s` replacement instead of `.format()` in log messages.

```python
log.debug('Refreshing pillar. Notify: %s', notify)
```

Reply:
@terminalmage , I fixed it.